js/ui/lookup.js

```diff
@@ -1129,6 +1087,28 @@ const Lookup = DropDownList.inherit({
             case 'cleanSearchOnOpening':
             case '_scrollToSelectedItemEnabled':
                 break;
+            case 'dropDownOptions':
+                switch(fullName) {
+                    case 'dropDownOptions.width':
+                        this._popupOptionChanged({
+                            name,
+                            fullName,
+                            value: value === 'auto' ? this.initialOption('dropDownOptions').width : value
+                        });
+                        this._options.cache('dropDownOptions', this.option('dropDownOptions'));
+                        break;
+                    case 'dropDownOptions.height':
+                        this._popupOptionChanged({
+                            name,
+                            fullName,
+                            value: value === 'auto' ? this.initialOption('dropDownOptions').height : value
+                        });
+                        this._options.cache('dropDownOptions', this.option('dropDownOptions'));
```

Comment:
```suggestion
                    case 'dropDownOptions.width':
                    case 'dropDownOptions.height':
                        this._popupOptionChanged({
                            name,
                            fullName,
                            value: value === 'auto' ? this.initialOption('dropDownOptions')[name] : value
                        });
                        this._options.cache('dropDownOptions', this.option('dropDownOptions'));
                        break;
```

Reply:
`name` equals 'dropDownOptions' here. I had idea to use something like `this.initialOption('dropDownOptions')[fullName.split('.')[1]]` but this looks realy weird :)

Reply:
implemented wia `getFieldName` utill